sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -347,11 +347,14 @@ def _handle_exception(self, exception: BaseException) -> "ServiceBusError":
             # failure modes, that motivates us to disallow this.
             if self._session and self._running and isinstance(error, (SessionLockLostError, ServiceBusConnectionError)):  # type: ignore
                 self._session._lock_lost = True  # type: ignore
-                self._close_handler()
```

Comment:
this close_handler() -- weird?

Reply:
The receiver will close itself in pyamqp now bc we get the error from the service - but still needed in pyamqp